examples/email-auth-local-storage/README.md

```diff
@@ -10,6 +10,8 @@ The overall flow for email auth is outlined below:
 
 This example contains an auth page as well and enables you to create wallets using an email auth credential. Unlike our other [email auth example](https://github.com/tkhq/sdk/tree/main/examples/email-auth), this does not use an iframe, which typically houses a target embedded key. Instead, a target embedded key is created within the application itself and stored in local storage.
 
+You can try this example quickly on Stackblitz -- follow the instructions below
```

Comment:
we can suggest jumping to Step 4 here (and we should be able to add a link to it via markdown)